addons/mrp/views/res_config_settings_views.xml

```diff
@@ -18,7 +18,6 @@
                                 </div>
                                 <div class="o_setting_right_pane" id="workorder_settings">
                                     <label for="group_mrp_routings" string="Work Orders"/>
-                                    <a href="https://www.odoo.com/documentation/user/14.0/manufacturing/management/manufacturing_order.html#manage-manufacturing-with-routings-and-work-centers" title="Documentation" class="o_doc_link" target="_blank"></a>
```

Comment:
I would at least replace by this one https://www.odoo.com/documentation/user/14.0/manufacturing/management/manufacturing_order.html#manage-manufacturing-with-routings-and-work-centers

Reply:
Isn't that the dead link I just removed?

Reply:
haha yes wrong copy-paste. This is the good link :  https://www.odoo.com/documentation/user/14.0/manufacturing/management/bill_configuration.html#adding-a-routing